annotate.el

```diff
@@ -115,6 +115,43 @@
   :type 'string
   :group 'annotate)
 
+(defcustom annotate-maximum-size-checksum 5000000
+  "Calculate checksum of  the current buffer only if  the size is
+less than this size (in characters)"
+  :type  'integer
+  :group 'annotate)
+
+(defconst annotate-warn-file-changed-control-string
+  (concat "The file '%s' has changed on disk "
+          "from the last time the annotations was saved. "
+          "Chances are that they will not be displayed correctly")
+  "The  message to  warn user  that  file has  been modified  and
+  annototatnions position could be outdated")
+
+(defcustom annotate-blacklist-major-mode '(org-mode)
+  "Prevent  loading of  annotated-mode  When  the visited  file's
+major mode is a member of this list (space separated entries)."
+  :type  '(repeat symbol)
+  :group 'annotate)
```

Comment:
This was discussed in a different issue as well. Do you know if the other issue made any progress on this?

Reply:
I still have problem with global mode but i think adding this variable was a good idea anyway because the incompatibility with org-mode is a known problem and this setting could prevent a bit of frustration for an user that do not read documentation with attention (like me for example)! ;-D

Reply:
OK, cool.